app/src/main/java/com/arctouch/codechallenge/details/DetailsFragment.java

```diff
@@ -0,0 +1,60 @@
+package com.arctouch.codechallenge.details;
+
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.text.TextUtils;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.arctouch.codechallenge.R;
+import com.arctouch.codechallenge.model.Movie;
+import com.arctouch.codechallenge.util.MovieImageUrlBuilder;
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.RequestOptions;
+
+/**
+ * Created by wanderleyfilho on 7/22/18.
+ */
+
+public class DetailsFragment extends Fragment {
+
+    public Movie movie;
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
+        View view =  inflater.inflate(R.layout.details_fragment, container, false);
+        final MovieImageUrlBuilder movieImageUrlBuilder = new MovieImageUrlBuilder();
+        setHasOptionsMenu(true);
+
+        ImageView posterImage = view.findViewById(R.id.imagePosterDetails);
+        TextView genreTextView = view.findViewById(R.id.textGenre);
+        TextView overviewTextView = view.findViewById(R.id.textOverview);
+        TextView releaseTextView = view.findViewById(R.id.textRelease);
+
+        if (TextUtils.isEmpty(movie.backdropPath) == false) {
```

Comment:
You have this same verification and code in your adapter to load the movie poster. Can you think of a way to get rid of this repetition?

Reply:
Yes, I could make the verification on the model level, so it would be in only one place